File: UPCODER/Cấu trúc dữ liệu/1. Tìm kiếm/TKTUANTU2.py
def sequentialSearch(arr, n, x):
    for i in range(0, n):
        if arr[i] == x:
            return i
    return -1
    
n, x = map(int, input().split())
arr = list(map(int, input().split()))
ans = sequentialSearch(arr, n, x)

if ans == -1:
    print(-1)
else:
    for i in range(0, n):
        if arr[i] == x:
            print(i, end = " ")

File: UPCODER/Cấu trúc dữ liệu/1. Tìm kiếm/TKTUANTU2.cpp
#include <bits/stdc++.h>
using namespace std;

int sequentialSearch(int arr[], int n, int x) {
    for (int i = 0; i < n; i++)
        if (arr[i] == x)
            return i;
    return -1;
}

int main() {
    int n, x;
    cin >> n >> x;
    
    int arr[n];
    for (int &i : arr) cin >> i;
    
    if (sequentialSearch(arr, n, x) != -1) {
        for (int i = 0; i < n; i++)
            if (arr[i] == x)
                cout << i << " ";
    } else cout << -1;
    
    return 0;
}